packages/textarea/src/LionTextarea.js

```diff
@@ -59,17 +59,18 @@ export class LionTextarea extends ObserverMixin(LionInput) {
     this.maxRows = 6;
   }
 
-  connectedCallback() {
-    // eslint-disable-next-line wc/guard-super-call
-    super.connectedCallback();
-    this.setTextareaMaxHeight();
-    autosize(this.inputElement);
-  }
-
   disconnectedCallback() {
     autosize.destroy(this.inputElement);
   }
 
+  async firstUpdated() {
+    if ('HTMLSlotElement' in window === false) {
```

Comment:
don't IE11 and Edge have this class from the webcomponents polyfills?

Reply:
Seems awkward right? I've thought it too, but It doesn't appear to be registering the slot element as HTMLSlotElement, at least I've doubled checked and it was not part of the window object after the polyfill is loaded